### PR TITLE
Tween nested object

### PR DESCRIPTION
**Pull request by talltyler**
There didn't seem to be enough love for https://github.com/tweenjs/tween.js/pull/313 to get it merged in. To move the ball forward I made a version based on the suggestion by @mikebolt to make this work without the string interpolation.

The performance impact should be a single function call for each level of nesting.

### CODE REVIEW

**Comment by mikebolt** [comment]
I think this is a great feature. I will review it soon.

**Comment by dalisoft** [comment]
Have you checked es6-tween? It uses my module called InterTween that does thing for your. Interpolates everything, just pass elapsed/value. Maybe es6-tween would be your choice?

**Comment by talltyler** [comment]
I like Tween.js and I would love to get this PR merged, but I will look more at es6-tween

**Comment by trusktr** [comment]
I like this idea too. From a basic skim, it looks good. Need to review one more time in more depth.

File: docs/user_guide.md [hunk 1]
```diff
@@ -390,6 +390,16 @@ relativeTween.start(); // Makes x go to -100 +100 = 0
 
 Check [09_relative_values](../examples/09_relative_values.html) for an example.
 
+### Tweening nested object
+
+Tween.js can also change properties across nested object. For example:
```
Comment (rtpHarry) [inline docs/user_guide.md]: across **a** nested object or across nested object**s**

**Comment by miltoncandelero** [comment]
Is this dead on the water?

**Comment by trusktr** [comment]
Sorry for the delays! I would like to get things rolling. Tracking this in the new pull request tracking board: https://github.com/tweenjs/tween.js/projects/4

I'd like to set up GitHub Actions next to get automated tests running again, so we can more easily test pull request changes and get them merged!

**Comment by trusktr** [comment]
Moved to #539 and merged. Thank you for this nice feature!

**Pull request closed without merging**